rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/TreeView.java

```diff
@@ -134,27 +136,24 @@ Canvas build() {
         this.tree = new Tree();
         tree.setModelType(TreeModelType.PARENT);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "nodeId" 5 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)